bcdaworker/worker/worker.go

```diff
@@ -164,20 +166,41 @@ func (w *worker) ProcessJob(ctx context.Context, job models.Job, jobArgs models.
 	return nil
 }
 
-func moveFiles(tempDir string, stagingDir string) error {
+func compressFiles(tempDir string, stagingDir string) error {
 	// Open the input file
 	files, err := os.ReadDir(tempDir)
 	if err != nil {
 		err = errors.Wrap(err, "Error reading from the staging directory for files for Job")
 		return err
 	}
+	gzipLevel, err := strconv.Atoi(os.Getenv("COMPRESSION_LEVEL"))
```

Comment:
Did we want to have a error thrown for when the compression level is < 1 or > 9 for feedback?

Reply:
Oop, sorry for the double-dip.